watch/inotify.go

```diff
@@ -134,3 +141,28 @@ func (fw *InotifyFileWatcher) ChangeEvents(t *tomb.Tomb, pos int64) (*FileChange
 
 	return changes, nil
 }
+
+func (fw *InotifyFileWatcher) checkAndNotifyIfModifiedInBetween(changes *FileChanges) (os.FileInfo, bool) {
+	fi, err := os.Stat(fw.Filename)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			util.Error("Failed to stat file %v: %v", fw.Filename, err)
+			// Treat it as a deleted file as we cannot read it anyway.
+		}
+		_ = RemoveWatch(fw.Filename)
+		changes.NotifyDeleted()
+		return nil, true
```

Comment:
until I looked up the usage I thought that the `bool` indicated whether the file was modified or not.

Reply:
It's if the file was deleted or not